main.go

```diff
@@ -68,26 +73,50 @@ func hello(c echo.Context) error {
 	return c.String(http.StatusOK, "Hello, World!")
 }
 
-// user
-// @Summary Add a new User
-// @Description Adds a new user to the list of Users
-// @ID add-user
+// OF Trace
+// @Summary Send trace request to OpenFisca
+// @ID of-trace
 // @Accept  json
 // @Produce  json
-// @Success 200 {string} string	"Returns new user"
-// @Router /user [post]
-func user(c echo.Context) error {
-	// Create a new user
-	u := new(User)
-	// Bind the request body to the new user
-	if err := c.Bind(u); err != nil {
-		return err
+// @Success 200 {string} string	"Returns OpenFisca trace response"
+// @Router /trace [post]
+func trace(c echo.Context) (err error) {
+
+	data := c.Request().Body
+
+	body, err := ioutil.ReadAll(data)
+	if err != nil {
+		log.Printf("Error reading body: %v", err)
+		// http.Error("can't read body", http.StatusBadRequest)
+		return
+	}
+
+	req, err := http.NewRequest("POST", "http://localhost:5000/trace", bytes.NewBuffer(body))
```

Comment:
We should create a new package and keep logic separate from the http request. A good idea would be to make one package for OF and we can then put any new code directly in there